src/parser/mod.rs

```diff
@@ -1274,6 +1274,50 @@ impl<'a> Parser<'a> {
         }
     }
 
+    pub fn parse_utility_options(&mut self) -> Result<Vec<(Ident, Option<String>)>, ParserError> {
```

Comment:
Can we add tests demonstrating the added functionality?

Reply:
Thank you. Added explain with option list tests.